src/endpoint/s3/ops/s3_put_bucket_lifecycle.js

```diff
@@ -111,6 +112,11 @@ async function put_bucket_lifecycle(req) {
         if (rule.Status?.length !== 1) {
             dbg.error('Rule should have status', rule);
             throw new S3Error(S3Error.InvalidArgument);
+        } else if (rule.Status?.length === 1) {
+            if (!s3_const.STATUS_VAL.ENUM.includes(rule.Status[0])) {
+                dbg.error('Rule should not have status value other than "Enabled" and "Disabled" ', rule);
```

Comment:
You can reuse the enum in the valid values printings.

Reply:
Done, Thanks